src/gerbil/compiler/driver.ss

```diff
@@ -87,6 +87,7 @@ namespace: gxc
 
   (let ((outdir      (pgetq output-dir: opts))
         (invoke-gsc? (pgetq invoke-gsc: opts))
+        (target      (pgetq target: opts))
```

Comment:
this needs to be passed to gsc too.

Reply:
What do you mean? Where should it be passed and how?

Reply:
when we invoke gsc, we need to tell it the target.

Reply:
You are mauling the gsc options in gxc's main, but this doesnt work prpgrammarically -- say through make.